File: official/docs/java/v5/billing/create-ep-credit-card.java
package referral;

import com.easypost.exception.EasyPostException;

public class Create {
    public static void main(String[] args) throws EasyPostException {
        EasyPost.apiKey = "EASYPOST_API_KEY";

        PaymentMethodObject creditCard = ReferralCustomer.addCreditCardToUser("REFERRAL_USER_API_KEY",
                "0123456789101234", 01, 2025, "111", PaymentMethod.Priority.PRIMARY);

        System.out.println(referralUser);
    }
}

File: official/docs/ruby/v5/billing/create-ep-credit-card.rb
require 'easypost'

client = EasyPost::Client.new(api_key: 'EASYPOST_API_KEY')

credit_card = client.referral_customer.add_credit_card(
  'REFERRAL_USER_API_KEY',
  '0123456789101234',
  '01',
  '2025',
  '111',
)

puts credit_card
